pkgs/by-name/at/attic-client/package.nix

```diff
@@ -32,6 +33,14 @@ rustPlatform.buildRustPackage {
     SystemConfiguration
   ]);
 
+  patches = [
+    # Part of https://github.com/zhaofengli/attic/pull/159
```

Comment:
- https://github.com/zhaofengli/attic/pull/159

This PR mentions that 24.05 doesn't have Nix 2.24. We should backport that in order to make situations like this less onerous -- and now that there's a proposal to make 2.24 the next stable, _not_ having it in 24.05 is pretty constraining.

Reply:
I've opened https://github.com/NixOS/nixpkgs/pull/336108 for this backport.